res/controllers/midi-components-0.0.js

```diff
@@ -467,9 +467,19 @@
             // For the first messages, disregard the LSB in case
             // the first LSB is received after the first MSB.
             if (this.MSB === undefined) {
-                this.max = 127;
+                // a flaw in the Pot API does not mandate consumers
+                // to supply an accurate max value when using
+                // this with non-7-bit inputs. We cannot detect
+                // that in the constructor so set the max
+                // appropriately here
+                if (this.max === Component.prototype.max) {
```

Comment:
It checks if the user did not specify a custom max value. If that is the case, the maximum is not `2^7-1` but `2^14-1` since we're in the MSB handler. If the user did specify a custom max value, then this was intentionally chosen to be correct. As explained in the comment, this is a workaround for the flawed API.